app/views/users/new.html.erb

```diff
@@ -1,32 +1,10 @@
-<% provide :title, t("sign_up") %>
-<h1><%= t("sign_up") %></h1>
+<% provide :title, t(:sign_up) %>
+<% provide(:button_text, t(:create_my_account)) %>
+<h1><%= t(:sign_up) %></h1>
 <div class="row">
   <div class="col-md-6 col-md-offset-3">
     <%= form_for @user, url: signup_path do |f| %>
-
-      <% if @user.errors.any? %>
-        <div id="error_explanation">
-          <div class="alert alert-danger">
-            <%= t "the_form_contains" %>
-            <%= pluralize @user.errors.count, "error" %>.
-          </div>
-          <ul>
-          <% @user.errors.full_messages.each do |msg| %>
-              <li><%= msg %></li>
-            <% end %>
-          </ul>
-        </div>
-      <% end %>
-
-      <%= f.label :name %>
-      <%= f.text_field :name %>
-      <%= f.label :email %>
-      <%= f.email_field :email %>
-      <%= f.label :password %>
-      <%= f.password_field :password %>
-      <%= f.label :password_confirmation, t("confirmation") %>
-      <%= f.password_field :password_confirmation %>
-      <%= f.submit t("create_my_account"), class: "btn btn-primary" %>
+    <%= render "form" %>
```

Comment:
ident please

Reply:
why i dont have the line 27,28,29 in my terminall, but in github i have this line.

![image](https://user-images.githubusercontent.com/87749692/131650354-8c097ab5-9822-41f0-811e-51accb191e93.png)